stdlib/3/time.pyi

```diff
@@ -5,10 +5,17 @@
 # see: http://nullege.com/codes/search?cq=time
 
 import sys
-from typing import Tuple, Union
+from typing import Any, NamedTuple, Tuple, Union
 from types import SimpleNamespace
 
-TimeTuple = Tuple[int, int, int, int, int, int, int, int, int]
+if sys.version_info >= (3, 3):
+    TimeTuple = Union[
```

Comment:
OK, filed https://github.com/python/mypy/issues/2509. PEP 484 in general is way under-specified.

I'm still hesitant to accept this, although a bit less so given that there are no APIs returning TimeTuple. (However, there's another definition of TimeTuple in datetime, and that *is* returned by some methods. However, those stubs should be changed to return struct_time.)